css/app.css

```diff
@@ -35,3 +35,28 @@ gaia-list .destructive {
 a.navigation *, a.action *, a.edit * {
   pointer-events: none;
 }
+
+gaia-list span.group {
+  text-transform: capitalize;
+  font-size: 0.8em;
```

Comment:
better use some rem ?

Also, it seems weird to me that the font is smaller than the subitems ? This is like a title after all ?